recipe/patches/0002-linux-patchelf.patch

```diff
@@ -0,0 +1,10 @@
+--- make-target-2.sh    2024-03-29 12:02:12.000000000 -0500
++++ make-target-2.sh    2024-04-28 19:10:54.914542271 -0500
+@@ -50,6 +50,7 @@
+     echo Unknown option \'"$1"\' to make-target-2
+     exit 1
+ fi
++patchelf --add-rpath $PREFIX/lib ./src/runtime/sbcl
```

Comment:
I am wondering a bit wether this could be done in the build script, after everything is done?

Reply:
If I remember correctly, because `sbcl` first bootstrap itself, then use that bootstrap to build itself, the build would fail in the middle.